usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereClient.js

```diff
@@ -364,11 +358,6 @@
             currentStrokeWidth = _this.getCurrentStrokeWidth();
             var isCurrentPointOnBoard = _this.maybeProjectPointOntoBoard(whiteBoardIntersectionData, true);
             if (event.isLeftButton) {
-                if (!isCurrentPointOnBoard) {
```

Comment:
I don't _think_ this is the right solution - I think you simply want to return early if `!isCurrentPointOnBoard`. No?

Reply:
Ack